src/azure-cli/azure/cli/command_modules/apim/_params.py

```diff
@@ -124,6 +124,7 @@ def load_arguments(self, _):
     with self.argument_context('apim api import') as c:
         c.argument('service_name', options_list=['--service-name', '-n'], help="The name of the api management service instance", id_part=None)
         c.argument('path', arg_group='API', help='Required. Relative URL uniquely identifying this API and all of its resource paths within the API Management service instance.', required=True)
+        c.argument('specification_format', arg_type=get_enum_type(ImportFormat), help='Specify the format of the imported API.', required=True)
```

Comment:
You don't need to state it's required. Cli will detect it by custom function signature.

```suggestion
        c.argument('specification_format', arg_type=get_enum_type(ImportFormat), help='Specify the format of the imported API.')
```